etc/geoserver/workspaces/MobiData-BW/park-api-db/park-api_car_lines/featuretype.xml

```diff
@@ -49,68 +49,144 @@
       <virtualTable>
         <name>park-api_car_lines</name>
         <sql>SELECT&#xd;
-    id,&#xd;
+    site.id,&#xd;
     NULL AS parking_site_id,&#xd;
     source_id,&#xd;
     original_uid,&#xd;
     name,&#xd;
     operator_name,&#xd;
     purpose,&#xd;
-    type :: text,&#xd;
     &apos;site&apos; as parking_object,&#xd;
+    public_url,&#xd;
+    photo_url,&#xd;
     address,&#xd;
+    description,&#xd;
+    has_lighting,&#xd;
+    fee_description,&#xd;
+    has_fee,&#xd;
+    park_and_ride_type,&#xd;
     has_realtime_data,&#xd;
     NULL AS realtime_status,&#xd;
-    capacity,&#xd;
+    site.type :: text,&#xd;
+    site.max_stay,&#xd;
+    site.capacity,&#xd;
+    CASE&#xd;
```

Comment:
@richardforsterNVBW In a follow-up PR, or at least a separate commit, can you remove the `&#xd;`s (`\r`)?